App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Relay from 'react-relay';
+import { render } from 'react-dom';
+import { Modal } from 'xb-common-lib';
+import AppRouter from './AppRouter';
+
+import './node_modules/xb-common-lib/dist/xbCommonLib.css';
```

Comment:
I think we won't need to go that deep to retrieve the library CSS, as it will be on the bundle, right?